cmake/OpenMWMacros.cmake

```diff
@@ -142,3 +142,12 @@ foreach (u ${ARGN})
 add_hdr (OPENCS ${dir} ${u})
 endforeach (u)
 endmacro (opencs_hdrs_noqt)
+
+macro (openmw_add_executable target)
+	add_executable(${target} ${ARGN})
+	if (MSVC)
+		if (CMAKE_VERSION VERSION_GREATER 3.8)
```

Comment:
Won't this only be true for CMake versions 3.9 and up? VS_DEBUGGER_WORKING_DIRECTORY was added in version 3.8.

Reply:
You're right. Apparently `VERSION_GREATER_EQUAL` hasn't been around for that long, either, so I'll have to use an `OR`.